source/quickstart.rst

```diff
@@ -10,7 +10,7 @@
 Quickstart
 ==========
 
-This section of our documentation allows users to easily install the Wazuh server, the Wazuh indexer, and the Wazuh dashboard on the same host using the unattended installation script. By running the automated script, you install and configure the Wazuh in just a few minutes.
+This documentation section allows users to easily install the Wazuh server, the Wazuh indexer, and the Wazuh dashboard on the same host using the unattended installation script. By running the automated script, you install and configure the Wazuh in just a few minutes.
```

Comment:
Changing the wording a little bit to avoid the verb "allow".

```suggestion
This section provides instructions to easily install the Wazuh server, the Wazuh indexer, and the Wazuh dashboard on the same host using the unattended installation script. By running the automated script, you install and configure the Wazuh in just a few minutes.
```